.github/workflows/pytest.yml

```diff
@@ -40,7 +40,7 @@ jobs:
 
     - name: Install
       run: |
-        conda install \
+        mamba install \
```

Comment:
Why switch from conda to mamba sorry, I don't understand

Reply:
just to make the dependency resolution faster. install was taking 20 minutes, and I'm impatient! 
https://github.com/duartegroup/autodE/actions/runs/5432666711/jobs/9879860668

(would be awesome if we could get rid of some of them..)